tests/test_app/urls.py

```diff
@@ -2,12 +2,17 @@
 from django.conf.urls.static import static
 from django.contrib import admin
 from django.conf.urls import url, include
-
+import django
+from distutils.version import StrictVersion
 admin.autodiscover()
 
+extra_url_kwargs = {'prefix': 'admin'} \
+    if StrictVersion(django.get_version()) < StrictVersion('1.7')\
+    else\
+    {}
```

Comment:
I like to avoid the use of `\`. But in this particular case, I recommend use _inlines_ when it can fit in one line. My recommendation is to use the following code:

```
if StrictVersion(django.get_version()) < StrictVersion('1.7'):
    extra_url_kwargs = {'prefix': 'admin'}
else:
    extra_url_kwargs = {}
```